src/rhalphalib/function.py

```diff
@@ -279,3 +342,7 @@ def parameters(self):
     @property
     def correlated_params(self):
         return self._correlated
+
+    @property
+    def correlated_str(self):
```

Comment:
Can this be `"\n".join(p.formula(rendering=True) for p in self._correlated)`?
Is it used anywhere or just for debugging?

Reply:
I had not realized there was a similar feature. Yes, this suffices.

(I was using this when trying to decorrelate parameters in analytic background fits (rather than rhalpha TF fits).)